skyplane/obj_store/object_store_interface.py

```diff
@@ -121,5 +121,12 @@ def create(region_tag: str, bucket: str):
 
             storage_account, container = bucket.split("/", 1)  # <storage_account>/<container>
             return AzureBlobInterface(storage_account, container)
+
+        elif region_tag.startswith("hdfs"):
+            print(f"Creating HDFS interface for bucket {bucket}")
+            from skyplane.obj_store.hdfs_interface import HDFSInterface
+
+            print(f"attme0ting to create hdfs bucket {bucket}")
```

Comment:
```suggestion
            print(f"attempting to create hdfs bucket {bucket}")
```

Reply:
Thanks for catching this @zizhong!

Reply:
No problem! Thanks for working on this! Looking forward to testing with our use cases.